core/test/qa_Block.cpp

```diff
@@ -263,7 +261,8 @@ const boost::ut::suite _stride_tests = [] {
         interpolation_decimation_test({ .n_samples =  100, .numerator =    101, .denominator =  101, .exp_in = 0 , .exp_out = 0, .exp_counter = 0 }, thread_pool);
 
         interpolation_decimation_test({ .n_samples =  100, .numerator = 5, .denominator = 11, .out_port_min = 10 , .out_port_max = 41, .exp_in = 88, .exp_out = 40, .exp_counter =  1 }, thread_pool);
-        interpolation_decimation_test({ .n_samples =  100, .numerator = 7, .denominator =  3, .out_port_min = 10 , .out_port_max = 10, .exp_in =  0, .exp_out =  0, .exp_counter =  0 }, thread_pool);
+// TODO: check with Semen if this test is possibly ill-defined and can never finish
+//        interpolation_decimation_test({ .n_samples =  100, .numerator = 7, .denominator =  3, .out_port_min = 10 , .out_port_max = 10, .exp_in =  0, .exp_out =  0, .exp_counter =  0 }, thread_pool);
```

Comment:
As discussed privately, the test was intentionally ill-defined. One needs to think how to unit-test ill-defined settings.

Reply:
OK. Then my assessment was correct. Have to see how/where we handle this case: block, scheduler, ...elsewhere/not at all? -> tomorrow's discussion.